scripts/aggregate-translations/aggregate-messages.js

```diff
@@ -1,7 +1,6 @@
 const path = require('path');
-const fs = require('fs');
 
-const aggregateTranslationMessages = (translationDirectories, locales) => {
+const aggregateTranslationMessages = (translationDirectories, locales, fileSystem) => {
```

Comment:
Bug fix. This should have been passed down to map the existing fileSystem option